src/components/CartCard/CartCard.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from "react";
+import "./CartCard.css";
+import AddIcon from "@mui/icons-material/Add";
+import RemoveIcon from "@mui/icons-material/Remove";
+import { useCart } from "../../context/CartContext";
+import { useWishlist } from "../../context/WishlistContext";
+
+export const CartCard = ({ _id, img, name, price, description, quantity }) => {
+  const [count, setCount] = useState(quantity);
+  const { cartDispatch } = useCart();
+  const { wishlistDispatch } = useWishlist();
+  // const finalPrice =
+
+  useEffect(() => {
+    count < 1 || isNaN(count) ? setCount(1) : setCount(count);
+    cartDispatch({
+      type: "CART_QUANTITY",
+      payload: { _id: _id, quantity: count },
+    });
+  }, [count]);
+
+  return (
+    <div className="card horizontal-card flex" key={_id}>
```

Comment:
the `key` property is not needed here, it is used where the `map` is being called